INSTALL.md

```diff
@@ -46,6 +46,12 @@ Make sure to have latest flex/bison in $PATH :
 export PATH=/usr/local/opt/flex/bin:/usr/local/opt/bison/bin:/usr/local/bin/:$PATH
 ```
 
+On Apple M1, corresponding brew paths are under `/opt/homebrew/opt/`:
+
+```sh
+export PATH=/opt/homebrew/opt/flex/bin:/opt/homebrew/opt/bison/bin:$PATH
```

Comment:
Is it best to be so specific. When I installed homebrew, I followed their recommendation of putting into the $HOME/.zprofile file the line
```
eval $(/opt/homebrew/bin/brew shellenv)
```
and I see in my PATH
```
/opt/homebrew/bin:/opt/homebrew/sbin
```
On my machine bison and flex are in /usr/bin and they both have the architecture
```x86_64 arm64e```. I assume they came from the command line tools install.  But I have to admit, my machine is no longer fresh and I have made many mistakes between fresh and now.

Reply:
Somehow  `/opt/homebrew/bin:/opt/homebrew/sbin` don't have flex/bison:

```
bb-fvff266jq05n:build_m1 kumbhar$ ls /opt/homebrew/bin/
2to3			gdbm_dump		lzcmp			msgattrib		msginit			pytest			xz
2to3-3.9		gdbm_load		lzdiff			msgcat			msgmerge		python3			xzcat
autopoint		gdbmtool		lzegrep			msgcmp			msgunfmt		python3-config		xzcmp
brew			gettext			lzfgrep			msgcomm			msguniq			python3.9		xzdec
ccmake			gettext.sh		lzgrep			msgconv			ngettext		python3.9-config	xzdiff
cmake			gettextize		lzless			msgen			pip3			recode-sr-latin		xzegrep
cpack			idle3			lzma			msgexec			pip3.9			unlzma			xzfgrep
ctest			idle3.9			lzmadec			msgfilter		py.test			unxz			xzgrep
easy_install-3.9	isympy			lzmainfo		msgfmt			pydoc3			wheel3			xzless
envsubst		lzcat			lzmore			msggrep			pydoc3.9		xgettext		xzmore

```

About `/usr/bin` : that has very ancient version:

```
bb-fvff266jq05n:build_m1 kumbhar$ /usr/bin/bison --version
bison (GNU Bison) 2.3
Written by Robert Corbett and Richard Stallman.

Copyright (C) 2006 Free Software Foundation, Inc.
```